doc/source/getting_started/write_code/index.rst

```diff
@@ -0,0 +1,101 @@
+.. _ref_write_code:
+
+============
+Develop code
+============
+
+You can help improve PyDPF-Core by fixing a bug. To do it, you must set up the repository
+on your local machine as per the following steps:
+
+- :ref:`ref_write_code_clone`
+- :ref:`ref_write_code_check_install`
+- :ref:`ref_write_code_develop_code`
+
+.. _ref_write_code_clone:
+
+Clone the repository
+--------------------
+
+Before cloning the PyMAPDL repository, you must install a version control system such as Git.
+
+Then, clone and install the latest version of PyDPF-Core in development mode (using ``pip`` with the ``-e``
+development flag) by running this code:
+
+.. code::
+
+    git clone https://github.com/ansys/pydpf-core
+    cd pydpf-core
+    pip install -e .
+
+.. _ref_write_code_check_install:
+
+Check the installation
+----------------------
+
+Run the following Python code to verify your PyDPF-Core installation:
+
+.. code::
+
+   from ansys.dpf.core import Model
+   from ansys.dpf.core import examples
+   model = Model(examples.find_simple_bar())
+   print(model)
+
+.. _ref_write_code_develop_code:
+
+Develop the PyDPF-Core code
```

Comment:
I think this can be found in the PyAnsys developer documentation, so I'd rather we redirect to what they provide.

Reply:
Coding style [here](https://dev.docs.pyansys.com/coding-style/index.html)
How to contribute [here](https://dev.docs.pyansys.com/how-to/contributing.html)